types/node/module.d.ts

```diff
@@ -334,14 +334,6 @@ declare module "module" {
         interface ImportAttributes extends NodeJS.Dict<string> {
             type?: string | undefined;
         }
-        type ModuleFormat =
-            | "builtin"
-            | "commonjs"
-            | "commonjs-typescript"
-            | "json"
-            | "module"
-            | "module-typescript"
-            | "wasm";
```

Comment:
This is an exported type, and removing it would be a breaking change. It can be left without any issue; even if it's not used internally, it may still be of use to consumers.

Reply:
OK; wasn't sure if I should keep it. I'll revert this.